src/redismodule.h

```diff
@@ -629,6 +646,11 @@ REDISMODULE_API int (*RedisModule_ZsetRangePrev)(RedisModuleKey *key) REDISMODUL
 REDISMODULE_API int (*RedisModule_ZsetRangeEndReached)(RedisModuleKey *key) REDISMODULE_ATTR;
 REDISMODULE_API int (*RedisModule_HashSet)(RedisModuleKey *key, int flags, ...) REDISMODULE_ATTR;
 REDISMODULE_API int (*RedisModule_HashGet)(RedisModuleKey *key, int flags, ...) REDISMODULE_ATTR;
+REDISMODULE_API int (*RedisModule_StreamAdd)(RedisModuleKey *key,
+                                             RedisModuleStreamID *id,
+                                             int flags,
+                                             RedisModuleString **argv,
+                                             int64_t numfields) REDISMODULE_ATTR;
```

Comment:
styling: i see that (maybe wrongly) none of the other APIs are broken to multiple lines in this file (although some are 300 chars long).
for now i think we should be consistent with that.

p.s. i doubt we really want `int64_t` for numfields, `long` would be better (for 32 bit builds)

Reply:
the code inside redis is more flexible to change. the API needs to be ABI compatible, so what we set now is there forever.
i think `long` is the right one here.